### ISSUE TITLE
Map pin active state is not removed until a different map pin is made active

### ISSUE THREAD

**Issue by cbessee** [spatialdev/react-seafood#65]
### To reproduce

Select a vendor booth on the map and observe that the pin is styled according the active state. Close the bottom panel and note that the active state still remains. Select a DIFFERENT map pin and note that the active state is removed before applying to the newly selected map pin. This causes a brief, distracting flash of the previously selected map pin

### Suggested fix
We should remove the pin active state when closing the bottom drawer rather than when selecting a new map pin

**Comment by danielduhh** [comment]
This only happens when selecting the close button in the bottom panel.

The deselect map icon logic exists in the SwipableDrawer onClose, but should also exist here:
https://github.com/spatialdev/react-seafood/blob/development/src/components/bottomDrawer/bottomDrawer.js#L121

**Comment by slindy23** [comment]
I disagree with @danielduhh on this. It happens every time I click on a map pin and I never touch the bottom panel.

**Comment by danielduhh** [comment]
There's two problems
1. the jumpy animation when you click a map pin
2. the active state doesn't get removed when selecting the close icon on the bottom panel
